src/test/java/com/amihaiemil/eoyaml/YamlIndentationTestCase.java

```diff
@@ -144,4 +147,32 @@ public void readsBadlyIndentedMapping() throws Exception {
             Matchers.equalTo("rultor")
         );
     }
+
+    /**
+     * A badly indented YAML sequence throws an exception.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void testIndentationAndYamlIndentationException() throws Exception {
+        try {
+            final List<YamlLine> lines = new ArrayList<>();
+            lines.add(new RtYamlLine("value1", 1));
+            lines.add(new RtYamlLine(" lGvSz", 2));
+            lines.add(new RtYamlLine("value3", 3));
+            final YamlLines yamlLines = new AllYamlLines(lines);
+            final YamlNode seq = yamlLines.toYamlNode(
+                    new RtYamlLine("foldedSequence:|-", 0), false);
```

Comment:
@lacinoire I think you can add this first line to the list and call toYamlNode without params. But I'm not sure anymore, to be honest

Reply:
I found no toYamlNode without any parameters in YamlLines / AllYamlLines.
So I'll leave it like this 🙂